pkg/controller/defaults/placements.go

```diff
@@ -104,29 +104,21 @@ var (
 		},
 
 		"rgw": rook.Placement{
-			NodeAffinity: &corev1.NodeAffinity{
```

Comment:
Why are we removing this?

Reply:
Can you check the answer provided by @leseb for a similar question
https://github.com/openshift/ocs-operator/pull/505/#issuecomment-628454073

Reply:
It explains why we have PodAntiAffinity. It does not explain why we removed NodeAffinity and Tolerations which are defaults.

Reply:
I have created the `Placement` according to the sample @leseb has provided in the doc. I had a chat with @leseb , he mentioned that the provided placement is the minimal / required sample, so I will add the `Toleration` and `NodeAffinity` (in addition to the changes)

Reply:
Added back the original `rgw` placement with changes required for the anti-affinity (into the `Changes for 'GatewaySpec', required for a multi-instances` commit)...
@umangachapagain , please take a look...